dalston/functions.php

```diff
@@ -206,7 +206,7 @@ function dalston_editor_styles() {
 
 	// Hide duplicate palette colors
 	$colors_array = get_theme_mod( 'colors_manager', array( 'colors' => true ) ); // color annotations array()
-	if ( ! empty( $colors_array ) && $colors_array['colors']['txt'] != '#1E1E1E' ) { // $config-global--color-foreground-light-default;
+	if ( ! empty( $colors_array ) && ! empty( $colors_array['colors'] ) && ! empty( $colors_array['colors']['txt'] ) && $colors_array['colors']['txt'] != '#1E1E1E' ) { // $config-global--color-foreground-light-default;
```

Comment:
It's not directly related to this feature. It just fixes a warning that I noticed in the editor.